aleph-client/src/pallets/author.rs

```diff
@@ -1,17 +1,16 @@
 use codec::Decode;
 
-use crate::{aleph_runtime::SessionKeys, Connection};
+use crate::{aleph_runtime::SessionKeys, ConnectionApi};
 
 #[async_trait::async_trait]
 pub trait AuthorRpc {
     async fn author_rotate_keys(&self) -> anyhow::Result<SessionKeys>;
 }
 
 #[async_trait::async_trait]
-impl AuthorRpc for Connection {
+impl<C: ConnectionApi> AuthorRpc for C {
```

Comment:
`AsConnection` should be enough here?

Reply:
Right, just wanted to be consistent

Reply:
Can change it if you think it's important